classes/components/listPanels/PKPSubmissionsListPanel.inc.php

```diff
@@ -0,0 +1,208 @@
+		);
+
+		$config['filters'] = [
+			array(
+				'filters' => array(
+					array(
+						'param' => 'isOverdue',
+						'value' => true,
+						'title' => __('common.overdue'),
+					),
+					array(
+						'param' => 'isIncomplete',
+						'value' => true,
+						'title' => __('submissions.incomplete'),
+					),
+				),
+			),
+			array(
+				'heading' => __('settings.roles.stages'),
+				'filters' => $this->getWorkflowStages(),
+			),
+		];
+
+		// Load grid localisation files
+		\AppLocale::requireComponents(LOCALE_COMPONENT_PKP_GRID);
+		\AppLocale::requireComponents(LOCALE_COMPONENT_PKP_SUBMISSION);
+
+		$config['i18n'] = array_merge($config['i18n'], [
+			'id' => __('common.id'),
+			'add' => __('submission.submit.newSubmissionSingle'),
+			'empty' => __('submission.list.empty'),
+			'loadMore' => __('grid.action.moreItems'),
+			'incomplete' => __('submissions.incomplete'),
+			'delete' => __('common.delete'),
+			'infoCenter' => __('submission.list.infoCenter'),
+			'yes' => __('common.yes'),
+			'no' => __('common.no'),
+			'deleting' => __('common.deleting'),
+			'currentStage' => __('submission.list.currentStage'),
+			'confirmDelete' => __('submission.list.confirmDelete'),
+			'responseDue' => __('submission.list.responseDue'),
+			'reviewDue' => __('submission.list.reviewDue'),
+			'reviewComplete' => __('submission.list.reviewComplete'),
+			'viewSubmission' => __('submission.list.viewSubmission'),
+			'reviewsCompleted' => __('submission.list.reviewsCompleted'),
+			'revisionsSubmitted' => __('submission.list.revisionsSubmitted'),
+			'copyeditsSubmitted' => __('submission.list.copyeditsSubmitted'),
+			'galleysCreated' => __('submission.list.galleysCreated'),
+			'filesPrepared' => __('submission.list.filesPrepared'),
+			'discussions' => __('submission.list.discussions'),
+			'assignEditor' => __('submission.list.assignEditor'),
+			'dualWorkflowLinks' => __('submission.list.dualWorkflowLinks'),
+			'reviewerWorkflowLink' => __('submission.list.reviewerWorkflowLink'),
+			'incompleteSubmissionNotice' => __('submission.list.incompleteSubmissionNotice'),
+			'viewMore' => __('list.viewMore'),
+			'viewLess' => __('list.viewLess'),
+			'paginationLabel' => __('common.pagination.label'),
+			'goToLabel' => __('common.pagination.goToPage'),
+			'pageLabel' => __('common.pageNumber'),
+			'nextPageLabel' => __('common.pagination.next'),
+			'previousPageLabel' => __('common.pagination.previous'),
+		]);
+
+		// Attach a CSRF token for post requests
+		$config['csrfToken'] = $request->getSession()->getCSRFToken();
+
+		// Provide required constants
+		import('lib.pkp.classes.submission.reviewRound.ReviewRound');
+		import('lib.pkp.classes.submission.reviewAssignment.ReviewAssignment');
+		import('lib.pkp.classes.services.PKPSubmissionService'); // STAGE_STATUS_SUBMISSION_UNASSIGNED
+		$templateMgr = \TemplateManager::getManager($request);
+		$templateMgr->setConstant('WORKFLOW_STAGE_ID_SUBMISSION');
```

Comment:
This might also want a helper with array syntax...

Reply:
:+1: I've changed it to accept an array instead of string. If anyone needs it for a single constant they can pass an array with one item.